lib/pdk/cli/exec.rb

```diff
@@ -217,6 +217,9 @@ def run_process_in_clean_env!
         def run_process!
           command_string = argv.join(' ')
           PDK.logger.debug(_("Executing '%{command}'") % { command: command_string })
+          if context == :module
+            PDK.logger.debug(_("GEM_HOME is '%{gem_home}' and GEM_PATH is '%{gem_path}'") % { gem_home: @process.environment['GEM_HOME'], gem_path: @process.environment['GEM_PATH'] })
```

Comment:
Hmm this should perhaps specify that these are the environment values for the command process